docs/sdks/uid2-sdk-ref-android.md

```diff
@@ -81,9 +81,14 @@ The binary is published on Sonatype:
 
 ## Minimum Requirements
 
-Minimum requirements for this SDK are as follows:
+To consume the binary package of this SDK in your app:
 
-- Android Studio version: v2023.3.1+
+- Minimum target Android version: 4.4+ / API 19+ (SDK) 5.0+
+
+
+To run the development app as mentioned in the [Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side#client-side-integration-example) or build binary from source code, minimum requirements are as follows:
+
+- Android Studio version: Check the Android Gradle Plugin (AGP) version required by the UID2 SDK specified in the [code repository](https://github.com/IABTechLab/uid2-android-sdk/blob/main/gradle/libs.versions.toml), and check [here](https://developer.android.com/build/releases/gradle-plugin) for the corresponding Android Studio version required
```

Comment:
- Android Studio version: Check the Android Gradle Plugin (AGP) version required by the UID2 SDK specified in the [code repository](https://github.com/IABTechLab/uid2-android-sdk/blob/main/gradle/libs.versions.toml), and check [the Android Gradle release notes](https://developer.android.com/build/releases/gradle-plugin) for the corresponding Android Studio version required.

Reply:
fixed